### PR TITLE
[BEAM-3866] Remove WindowedValue on PCollections for Go SDK

### PR DESCRIPTION
**Pull request by herohde**
This is what the Model pipeline expects, although Dataflow expects the old way.

### CODE REVIEW

**Comment by herohde** [comment]
R: @lostluck @aljoscha 

This PR resolves one of the issues found with Go on Flink.

File: sdks/go/pkg/beam/core/graph/edge.go [hunk 1]
```diff
@@ -342,19 +342,18 @@ func NewCombine(g *Graph, s *Scope, u *CombineFn, in *Node) (*MultiEdge, error)
 		// However, the outbound type will be W<KV<A,O>> (where O is the output
```
Comment (lostluck) [inline sdks/go/pkg/beam/core/graph/edge.go]: Are the W<>s in these paragraphs left intentionally?

Comment (herohde) [reply sdks/go/pkg/beam/core/graph/edge.go]: Nope. Good catch!

**Comment by lostluck** [comment]
I suspect you may have done this already but running 
grep -r "W<" .
In your github.com/apache/beam/sdks/go directory should root out any files that have been missed.
From my own run I think the only file missing is sdks/go/README.md

**Pull request merged**